test/test_runner.go

```diff
@@ -225,13 +232,20 @@ func recoverPanics(onError func(error)) {
 }
 
 func (r *TestRunner) parseCheckAndInterpret(script string) (*interpreter.Program, *interpreter.Interpreter, error) {
-	env := runtime.NewBaseInterpreterEnvironment(runtime.Config{})
+	config := runtime.Config{
+		CoverageReportingEnabled: r.coverageReport != nil,
+	}
+	env := runtime.NewBaseInterpreterEnvironment(config)
 
 	ctx := runtime.Context{
 		Interface:   newScriptEnvironment(),
 		Location:    testScriptLocation,
 		Environment: env,
 	}
+	if r.coverageReport != nil {
+		r.coverageReport.ExcludeLocation(testScriptLocation)
```

Comment:
We will probably need to exclude this location also

```go
r.coverageReport.ExcludeLocation(stdlib.TestContractLocation)
```
to avoid "noise" for integration tests that import the built-in `Test` contract.

Reply:
Sounds good! Fixed in https://github.com/onflow/cadence-tools/pull/78/commits/4706ae1b9227f2ec5839f8be402370e122624f77